public/index.html

```diff
@@ -0,0 +1,741 @@
+                        <p>Director: BoxApp Media inc.</p>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div> 
+            <div class="pager"></div>
+          </div>
+        </div>
+      </section>
+      <section class="c1-screenshots" id="screens">
+        <div class="underlined-heading-screenshot">
+          <h1 class="underlined-heading-screenshot-text">Screenshots</h1>
+          <div class="underlined-heading-screenshot-underline"></div>
+        </div>
+        <div class="slider-outer">
+          <div class="slider-inner">
+            <img class="c1-image-shown" src="./img/cell1.png" alt=" Black iPhone showing different screens">
+            <img class="c1-image-hidden" src="./img/cell2.png" alt=" Black iPhones showing different screens">
+            <img class="c1-image-hidden" src="./img/cell3.png" alt=" Black iPhones showing different screens">
+          </div>
+          <div class="left-right">
+            <img class="c1-left" src="./img/left-arrow.png" alt="">
+            <img class="c1-right" src="./img/right-arrow.png" alt="">
+          </div>
+          </div>
+            <div class="slider-outer-2">
+              <div class="slider-inner-2">
+                <img class="c1-image-shown" src="./img/2-cells-dark.png" alt=" Black iPhone showing different screens">
+                <img class="c1-image-hidden" src="https://i.imgur.com/9002ywb.png" alt=" Black iPhones showing different screens">
+                <img class="c1-image-hidden" src="./img/2-cells-dark.png" alt=" Black iPhones showing different screens">
+              </div> 
+              <div class="left-right">
+                <img class="c1-left-2" src="./img/left-arrow.png" alt="">
+                <img class="c1-right-2" src="./img/right-arrow.png" alt="">
+              </div>
+            </div>
+            <div class="slider-outer-3">
+              <div class="slider-inner-3">
+                <img class="c1-image-shown" src="http://i.imgur.com/qUkuOdp.png" alt=" Black iPhone showing different screens">
+                <img class="c1-image-hidden" src="https://i.imgur.com/9002ywb.png" alt=" Black iPhones showing different screens">
+                <img class="c1-image-hidden" src="http://i.imgur.com/qUkuOdp.png" alt=" Black iPhones showing different screens">
+              </div>
+              <div class="left-right">
+                <img class="c1-left-3" src="./img/left-arrow.png" alt="">
+                <img class="c1-right-3" src="./img/right-arrow.png" alt="">
+              </div>
+            </div>
+          </section>
+          <section class="d1-download-more-apps" id="blog">
+            <div class="underlined-heading-download">
+              <h1 class="underlined-heading-download-text">More Apps by AppBox</h1>
+              <div class="underlined-heading-download-underline"></div>
+            </div>
+            <div class="carousel-2">
```

Comment:
I really enjoy the animated enhancement you've used here; it's a unique take on a common problem (showing multiple sets of different items) and it's pretty cleanly implemented.